configs/hugetlb.go

```diff
@@ -0,0 +1,9 @@
+package configs
+
+type Hugetlb struct {
```

Comment:
We should still put it in the cgroup file, the divisions of files there are very broad: cgroup, network, namespaces. This one is too fine to be in its own file.